### PR TITLE
Nicholas Carrigan Added to moderators section

### PR DESCRIPTION
**Pull request by JatinAgrawal94**
Screenshot (20)

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -226,6 +226,14 @@ export const moderators = [
     github: 'AllanRegush',
     badges: ['community'],
   },
+  {
+    name:'Nicholas Carrigan',
+    description:'Developer @freecodecamp',
+    website:'https://www.nhcarrigan.com',
+    imageSrc:'https://avatars.githubusercontent.com/u/63889819?s=400&u=9dab24c783a82afce97cfe6438f242e6df641243&v=4',
+    github:'https://github.com/nhcarrigan',
```
Comment (AllanRegush) [inline src/data.js:234]: By looking at the other Entries I think it should be like this:

```suggestion
    github:'nhcarrigan',
```

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1: Just a few adjustments.

**Comment by JatinAgrawal94** [comment]
Github link modified for moderator with username 'nhcarrigan'

**Comment by eddiejaoude** [comment]
Thank you all, great collaboration 💪

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**